elasticdl/pkg/ps/server.go

```diff
@@ -153,8 +159,9 @@ func (s *Server) Run(address string, serverDone chan bool) *grpc.Server {
 	if err != nil {
 		log.Fatalf("failed to start PS: %v", err)
 	}
-	// TODO: set maxReceiveMessageSize (default is 4M, too small for elasticdl), maxConcurrentStreams
-	grpcServer := grpc.NewServer()
+	grpcServer := grpc.NewServer(grpc.MaxRecvMsgSize(maxReceiveMessageLength),
+		grpc.MaxSendMsgSize(maxSendMessageLength),
+		grpc.MaxConcurrentStreams(maxConcurrentStreams))
```

Comment:
MaxConcurrentStreams can be set as the number of workers.

Reply:
Done